NAMING.txt

```diff
@@ -44,3 +44,7 @@ SleepSupplementsCorrelationsTests
 ProductivityLogAggregatesView
 SleepAggregatesView
 SleepAveragesView
+
+# try to have the full django model in the name, easier to decide on that pattern
+# make the resource ... plural
+ProductivityLogsSupplementsCorrelationsView
```

Comment:
Naming is hard, yo. Not sure how to name querysets

should they be ... supplements_queryset or supplement_queryset?